x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/mock.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IntegrationDetails } from './utils';
+import {
+  InstalledIntegrationArray,
+  RelatedIntegrationArray,
+} from '../../../../../common/detection_engine/schemas/common';
+
+export const relatedIntegrations: RelatedIntegrationArray = [
+  {
+    package: 'system',
+    version: '1.6.4',
+  },
+  {
+    package: 'aws',
+    integration: 'cloudtrail',
+    version: '~1.11.0',
+  },
+];
```

Comment:
I think we should keep all this mock data near each type (like `RelatedIntegrationArray`) definition. I can move it there in a follow-up PR. I should have added some mock data in https://github.com/elastic/kibana/pull/132409